### ISSUE TITLE
DSC_estimated_error_rates

### ISSUE THREAD

**Issue by gevro** [cancerit/NanoSeq#47]
Hi,
For most of our samples, DSC_estimated_error_rates are in the 1-4 e-10 range.
But a few DNA samples with more damage are ~10-fold higher: 1-2 e-9.

This is almost exclusively for C>A and C>T mutations.

What is an acceptable range for DSC estimated error rates?

Does 10-fold higher DSC_estimated_error_rate matter when it is still only 1-2e-9, which is ~100-fold lower than true signal that is in the e-7 range?

Thanks.

**Comment by fa8sanger** [comment]
That’s right, 1e-9 error rate is no problem if you are working on the e-7 range.

The DSC error rate goes up if DNA damage during library preparation increases. Oxidation during PCR can be particularly bad.

On 22 Oct 2022, at 15:37, gevro ***@***.******@***.***>> wrote:



Hi,
For most of our samples, DSC_estimated_error_rates are in the 1-4 e-10 range.
But a few DNA samples with more damage are ~10-fold higher: 1-2 e-9.

This is almost exclusively for C>A and C>T mutations.

What is an acceptable range for DSC estimated error rates?

Does 10-fold higher DSC_estimated_error_rate matter when it is still only 1-2e-9, which is ~100-fold lower than true signal that is in the e-7 range?

Thanks.

—
Reply to this email directly, view it on GitHub [github.com]<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_cancerit_NanoSeq_issues_47&d=DwMCaQ&c=D7ByGjS34AllFgecYw0iC6Zq7qlm8uclZFI0SqQnqBo&r=v9-R7fUmjpv-9Zaqyk1nlnlOC3qPkTEJz5tyYxg2uec&m=rAwEt8mw0SI9yBqk-DYR1BMot6h1dVZjfPpSQJ-NFxJfBnIwH7LQ8xhK4Y2XTBv4&s=loECsMp1ZOxkz8WinV-GV16rx9Ie1-MY8476OdigkK0&e=>, or unsubscribe [github.com]<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_ADNUT3MRJNPXMH55XOCT7J3WEP3ZVANCNFSM6AAAAAARL34FC4&d=DwMCaQ&c=D7ByGjS34AllFgecYw0iC6Zq7qlm8uclZFI0SqQnqBo&r=v9-R7fUmjpv-9Zaqyk1nlnlOC3qPkTEJz5tyYxg2uec&m=rAwEt8mw0SI9yBqk-DYR1BMot6h1dVZjfPpSQJ-NFxJfBnIwH7LQ8xhK4Y2XTBv4&s=aox0AhB2_gBsN7L_fWMLU4CdGw8yVEAb2-IadVGQMe4&e=>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>




-- 
 The Wellcome Sanger Institute is operated by Genome Research 
 Limited, a charity registered in England with number 1021457 and a 
 company registered in England with number 2742969, whose registered 
 office is 215 Euston Road, London, NW1 2BE.
